nym-vpn-core/crates/nym-vpnd/src/command_interface/listener.rs

```diff
@@ -901,11 +901,19 @@ impl NymVpnd for CommandInterface {
         &self,
         _: tonic::Request<()>,
     ) -> Result<tonic::Response<GetLogPathResponse>, tonic::Status> {
-        let dir = service::log_dir();
-        tracing::debug!("log file path: {}", dir.display());
+        let result = CommandInterfaceConnectionHandler::new(self.vpn_command_tx.clone())
+            .handle_get_log_path()
+            .await?;
+        let log_path = if let Some(path) = result {
+            path
+        } else {
+            tracing::info!("log path not set, fallback to default");
+            LogPath::default()
+        };
+        tracing::debug!("log dir path: {}", log_path.dir.display());
         Ok(tonic::Response::new(GetLogPathResponse {
-            path: dir.to_string_lossy().to_string(),
-            filename: service::DEFAULT_LOG_FILE.to_string(),
+            path: log_path.dir.to_string_lossy().to_string(),
+            filename: log_path.filename.to_string(),
```

Comment:
I would suggest implementing a conversion from `LogPath` to `GetLogPathResponse` instead. This how we have been doing it for new code for awhile now.